CHANGES.md

```diff
@@ -26,3 +26,5 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Ensured that marathon and SDK labeled reservations are not offered to other schedulers (D2IQ-68800)
 
 * Updated OpenResty to 1.15.8.4. (DCOS_OSS-5967)
+
+* Update Telegrapf configuration to reduce used resources on statsd inputs. (COPS-5629)
```

Comment:
spelling is Telegraf

`reduce used resources on statsd inputs` - for the release notes, can we relate this more to user experience (e.g. reduce errors, vary requests to reduce load, sample less frequently)